lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -51,36 +74,45 @@ template <class UfeValueType, class UsdValueType>
 class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
 {
 public:
+    typedef std::shared_ptr<UsdPointInstanceBatch<UsdValueType>> Batch;
```

Comment:
Based on modern C++ guidelines, we should switch to alias-declaration (`using` keyword) instead of `typedef`.